.eslintrc.yml

```diff
@@ -82,3 +84,6 @@ rules:
   # Chai friendly.
   no-unused-expressions: 'off'
   chai-friendly/no-unused-expressions: 'error'
+
+  # Disable some rules from jsdoc/recommended.
+  jsdoc/require-jsdoc: 0
```

Comment:
You could set this to `warn` which would give us results in CI while this is open.